react-ui/src/pages/entry/EntryModal.jsx

```diff
@@ -160,6 +161,11 @@ function EntryModal() {
   const handleKeyUp = ({ key }) => {
     let indexChange = 0;
     if (key >= '0' && key <= '5') {
+      // ??? I would love it if we could track whether votes mobile or desktop
+      // ??? and if desktop, how many were keyed
+      // https://stackoverflow.com/questions/39435395/reactjs-how-to-determine-if-the-application-is-being-viewed-on-mobile-or-deskto
+      changeRating(parseInt(key, 10), entry?.rating, true);
+      indexChange = 1;
       return;
```

Comment:
By returning here, the `indexChange` never gets used. Perhaps wrap the `switch` in an `else` clause and delete the `return`.

Also, is there a scenario where someone is working backwards for some reason and ends up having to click back twice? Maybe we should add an issue to change this setting, i.e. have auto-next be the default but at least allow the user to toggle it off?

Reply:
I moved the if statement into the switch default because it seemed a bit cleaner than the else. Another option is to just add `handleNavigate` into the `if` statement before the `return`.

I created an issue, but I'd imaging >99% of users will follow forward progression, and if you are going backwards than double-clicking back is manageable.

Reply:
Yeah, putting it in the default case makes sense.

And agreed, it's a low priority issue, but maybe we add it sometime down the line.